src/main/resources/config/liquibase/changelog/20210218130721_add_silo_logo.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.8.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="kdondziak" id="1613653681560-6">
+        <addColumn tableName="silo">
+            <column name="logo_base64" type="clob"/>
```

Comment:
Use:
`<property name="clobType" value="clob" dbms="oracle, h2, postgresql"/>`
`<property name="clobType" value="longtext" dbms="mysql"/>`

and then: `type="${clobType}"`